extensions/smallrye-jwt/deployment/src/test/java/io/quarkus/jwt/test/dev/SmallryeJwtProcessorDevModeTest.java

```diff
@@ -0,0 +1,77 @@
+package io.quarkus.jwt.test.dev;
+
+import jakarta.annotation.security.PermitAll;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.jwt.Claims;
+import org.hamcrest.Matchers;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.jwt.test.GreetingResource;
+import io.quarkus.test.QuarkusDevModeTest;
+import io.restassured.RestAssured;
+import io.restassured.http.Header;
+import io.smallrye.jwt.build.Jwt;
+
+public class SmallryeJwtProcessorDevModeTest {
+
+    @RegisterExtension
+    static QuarkusDevModeTest devMode = new QuarkusDevModeTest()
+            .withApplicationRoot((jar) -> jar.addClasses(
+                    GreetingResource.class, TokenResource.class).addAsResource(
+                            new StringAsset(""), "application.properties"));
+
+    @Test
+    public void shouldNotBeNecessaryToAddSignKeysOnApplicationProperties() {
+        String token = RestAssured.given()
```

Comment:
@mcruzdev Can you now try to remove `TokenResource` and use `JwtTokenKeys` ?

Reply:
@mcruzdev Hmm, you may be right, for the tests running in the `deployments`, the custom factory is needed, we'll use `JwtTokenKeys` in the integration tests

Reply:
@mcruzdev Sure, what about the devmode test though, it still uses a `TokenResource` ?

Reply:
@mcruzdev Sorry, I may be a bit confused. This test acquires a token from the `TokenResource`, can you please update the test not to use `TokenResource` ?